types/index.d.ts

```diff
@@ -16,179 +16,174 @@ import {
   GraphQLTypeResolver,
 } from 'graphql';
 
-export = graphqlHTTP;
+export {};
 
 type Request = IncomingMessage;
 type Response = ServerResponse & { json?: (data: unknown) => void };
 
-declare namespace graphqlHTTP {
+/**
+ * Used to configure the graphqlHTTP middleware by providing a schema
```

Comment:
So it just indentation changes you didn't change anything inside, right?

Reply:
Removed the namespace and exported the individual types